x/bank/atlas.toml

```diff
@@ -0,0 +1,34 @@
+[module]
+description = "The bank module is responsible for handling multi-asset coin transfers between accounts and tracking special-case pseudo-transfers which must work differently with particular kinds of accounts."
+homepage = "https://github.com/cosmos/cosmos-sdk"
+keywords = [
+  "tokens",
+  "bank",
+  "transfer",
+  "asset",
+  "fungible",
+  "coins",
+]
+
+name = "x/bank"
+
+[bug_tracker]
+url = "https://github.com/cosmos/cosmos-sdk/issues"
+
+[[authors]]
+name = "alexanderbez"
+
+[[authors]]
+name = "fedekunze"
+
+[[authors]]
+name = "cwgoes"
+
+[[authors]]
+name = "alessio"
+
+[version]
+documentation = "https://raw.githubusercontent.com/cosmos/cosmos-sdk/v0.39.1/x/bank/spec/README.md"
+repo = "https://github.com/cosmos/cosmos-sdk/releases/tag/v0.39.1"
+sdk_compat = "v0.39.x"
+version = "v0.39.1"
```

Comment:
Being that this is completely arbitrary, although we encourage semver, how should we proceed with versioning modules? The initial version I plan to publish of all the SDK modules will be those in 0.39.1 (Launchpad).

Do we want to stick with the SDK version for the SDK modules? Do we want to start at some arbitrary semver (e.g. v0.0.1)?

/cc @sunnya97 @aaronc @amaurymartiny @alessio

Reply:
I'd go for same version as SDK's

Reply:
For now our story of "can sdk modules have their own version"/"can they also be go modules"/"in which repo do they live" isn't quite clear yet, so I'd also go for same version as SDK's

Reply:
Personally, I'd prefer to go for a different version than the SDK, because that would allow us to to update them more quickly than the SDK.

Reply:
It seems you're outnumbered here @sunnya97. In the long-term I do agree with you, but it's not clear what version to use and what version of the module to start at.